### PR TITLE
fix: remove unneeded fields from result schema

### PR DESCRIPTION
**Pull request by carissa-tang**
## Notion ticket link

[Remove gradingStatus, score, and breakdown from Result](https://www.notion.so/uwblueprintexecs/Remove-gradingStatus-score-and-breakdown-from-Result-eba4af0e5b7c4f5cbb4a5fab712b6577?pvs=4)

## Implementation description
* Remove `gradingStatus` -> will always be "graded" when in the database
* Remove `score` and `breakdown` from ResultRequestDTO -> will always be null / empty
   * `Score` in model and response will always be non-null

Fixes missed from previous ticket changes
* Change `answers` type from `NumberOrArrayOrNull` to `[[Float]]` in graphql
* Remove `results` from `TestSessionRequestDTO` in graphql 
* Add missing interface docs for `createTestSessionResult`

## Steps to test
1. yarn test and tested all existing resolvers

## What should reviewers focus on?
* No breaking changes

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/services/implementations/testSessionService.ts [hunk 1]
```diff
@@ -176,6 +173,29 @@ class TestSessionService implements ITestSessionService {
     return testSessionDtos;
   }
 
+  async getTestSessionsBySchoolId(
```
Comment (carissa-tang) [inline backend/services/implementations/testSessionService.ts:176]: reordered to match interface's order

File: backend/services/implementations/testSessionService.ts [hunk 2]
```diff
@@ -226,48 +246,6 @@ class TestSessionService implements ITestSessionService {
     return testSessionDtos;
   }
 
-  private async mapTestSessionsToTestSessionDTOs(
```
Comment (carissa-tang) [inline backend/services/implementations/testSessionService.ts:229]: reordered ^

File: backend/services/implementations/testSessionService.ts (hunk 1 above)
Comment (joyce-shi) [reply backend/services/implementations/testSessionService.ts:176]: we love that

**Review by joyce-shi — APPROVED** [review]
LGTM! as a side note, we should probably update our endpoint & DB documentation some time soon haha

**Pull request merged**